sdk/python/kfp/__init__.py

```diff
@@ -16,9 +16,13 @@
 # https://packaging.python.org/guides/packaging-namespace-packages/#pkgutil-style-namespace-packages
 __path__ = __import__("pkgutil").extend_path(__path__, __name__)
 
-__version__ = '2.0.0b0'
+__version__ = '2.0.0-alpha.0'
```

Comment:
**Not this line of code**

Should Copyright on this file be 2022 because we are presently modifying it?

Reply:
Done.